frontend/src/components/ui/PropertiesDrawer/PropertiesDrawer.tsx

```diff
@@ -88,9 +88,12 @@ export default function PropertiesDrawer({
 
   const { fileBrowserState } = useFileBrowserContext();
   const { pathPreference, areDataLinksAutomatic } = usePreferencesContext();
-  const { ticket } = useTicketContext();
-  const { proxiedPath, dataUrl } = useProxiedPathContext();
+  const { ticketByPathQuery } = useTicketContext();
+  const { proxiedPathByFspAndPathQuery } = useProxiedPathContext();
   const { externalDataUrl } = useExternalBucketContext();
+
+  const ticket = ticketByPathQuery.data;
+  const proxiedPath = proxiedPathByFspAndPathQuery.data;
```

Comment:
How are you making sure that the .data is loaded for these. I would expect to see this use the isPending state to decide if the data has been loaded. I didn't notice an isPending check in the TicketsContext that would prevent this component from loading and passing the incorrect ticket value to TicketDetails.